www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -351,30 +329,20 @@
                 <HttpTlsTrustedCaCerts type="CertificateField">
                     <Type>ca</Type>
                 </HttpTlsTrustedCaCerts>
-                <HttpTlsServerName type="HostnameField">
-                    <ValidationMessage>Please enter a valid hostname or IP address.</ValidationMessage>
-                    <IpAllowed>Y</IpAllowed>
-                    <HostWildcardAllowed>Y</HostWildcardAllowed>
-                    <FqdnWildcardAllowed>Y</FqdnWildcardAllowed>
-                    <ZoneRootAllowed>N</ZoneRootAllowed>
-                </HttpTlsServerName>
+                <HttpTlsServerName type="HostnameField"/>
                 <description type="DescriptionField">
                     <Required>Y</Required>
                 </description>
             </handle>
             <accesslist type="ArrayField">
-                <accesslistName type="TextField">
+                <accesslistName type="DescriptionField">
                     <Required>Y</Required>
-                    <Mask>/^([\t\n\v\f\r 0-9a-zA-Z.,_*-\x{00A0}-\x{FFFF}]){1,255}$/u</Mask>
-                <ValidationMessage>Please provide a valid Access List Name.</ValidationMessage>
                 </accesslistName>
                 <clientIps type="NetworkField">
                     <Required>Y</Required>
-                    <NetMaskAllowed>Y</NetMaskAllowed>
                     <FieldSeparator>,</FieldSeparator>
                     <AsList>Y</AsList>
                     <Strict>Y</Strict>
-                    <ValidationMessage>Please enter valid IP address(es) or network(s), separated by commas.</ValidationMessage>
```

Comment:
Since the field is required, this validation message is a bit more helpful than the default since you allow multiple

Reply:
Thanks, I fixed this.